frontend/src/container/AdminAccount.jsx

```diff
@@ -219,6 +219,18 @@ class Account extends React.Component {
 
   handleChangeTimezone = newTimezone => console.log('(NYI) new timezone : ', newTimezone)
 
+  // INFO - GB - 2019-06-06 - This function is needed because i18next do not handle special characters and the phrase order change depending of the language
+  // https://github.com/tracim/tracim/issues/1847
+  setTitleAccordingToLang = () => {
```

Comment:
This solution works but it would be better to fix the charset issue.
Please do some research on where does this charset bug comes from